### PR TITLE
Send ExactTarget signups to sawmill for processing

### PR DESCRIPTION
**Pull request by cadecairos**
mozilla/donate.mozilla.org#771
mozilla/sawmill#62

@jbuck R?

### CODE REVIEW

File: routes/signup.js [hunk 1]
```diff
@@ -1,9 +1,8 @@
...
+var hatchet = require('hatchet');
+var url = process.env.SIGNUP;
 
-module.exports = function(transaction, callback) {
-  var url = process.env.SIGNUP;
-  callback = callback || function() {};
-  httpRequest.post({
+module.exports = function(transaction) {
+  hatchet.send("sign_up_for_mofo_newsletter", {
     url: url,
```
Comment (jbuck) [inline routes/signup.js]: If you make the sawmill worker more generic, you'll need to add the `method: 'POST'` key here

File: routes/signup.js [hunk 2]
```diff
@@ -15,5 +14,5 @@ module.exports = function(transaction, callback) {
       email: transaction.email,
       country: transaction.country
     }
-  }, callback);
+  });
```
Comment (jbuck) [inline routes/signup.js]: I'd leave the callback here to see whether the hatchet queue works or not

Comment (cadecairos) [inline routes/signup.js]: and what do we do if it doesn't?

Comment (cadecairos) [inline routes/signup.js]: okay, fixed up

File: routes/signup.js [hunk 3]
```diff
@@ -1,9 +1,8 @@
-var httpRequest = require('request');
+var hatchet = require('hatchet');
+var url = process.env.SIGNUP;
 
 module.exports = function(transaction, callback) {
-  var url = process.env.SIGNUP;
-  callback = callback || function() {};
-  httpRequest.post({
+  hatchet.send("send_post_request", {
     url: url,
```
Comment (jbuck) [inline routes/signup.js]: you'll need to add the method key here too

Comment (jbuck) [inline routes/signup.js]: oh wait, send post request

Comment (jbuck) [inline routes/signup.js]: f it, might as well add it anyways for super generic!

**Comment by jbuck** [comment]
@cadecairos r+

**Pull request merged**